src/check_fastq.py

```diff
@@ -27,6 +27,7 @@ def main():
         keys = ['s3://hca-util-upload-area/' + str(s3_object.key) for s3_object in my_bucket.objects.all()]
```

Comment:
Now to the lines:

L27-30 can be resumed to 1 line:
`keys = [f's3://hca-util-upload-area/{s3_object.key}' for s3_object in my_bucket.objects.all().filter(Prefix=uuid) if s3_object.key.endswith('.fastq.gz')]`

Also if using this, no need to check if uuid ends with `/` or not